LBHFSSPublicAPI/V1/Domain/TaxonomyEntity.cs

```diff
@@ -6,7 +6,8 @@ public class TaxonomyEntity
     {
         public int Id { get; set; }
         public string Name { get; set; }
-        public int ParentId { get; set; }
+        public int? ParentId { get; set; }
+        public string Description { get; set; }
```

Comment:
Note: So Martin knows about the about the "Description" change, but did we tell there was a change to the ParentId to make it nullable? Does this matter to his schema?

Reply:
I don't think it matters in the schema.  The reason I made it nullable is that I don't think we expect all taxonomies to have a parent as there might be some who are top level taxonomies (no parent).